asab/api/service.py

```diff
@@ -13,15 +15,33 @@
 
 ##
 
+asab.Config.add_defaults({
+	"general": {
+		"manifest": "/MANIFEST.json",
```

Comment:
@ateska maybe this should just be empty string ?

Reply:
yes